readme.md

```diff
@@ -33,6 +33,16 @@ pip3 install -r Requirements.txt
 python3 setup.py install
 ```
 
+## Quick and dirty Debian (and Ubuntu?) package
+
+This requires [stdeb](https://github.com/astraw/stdeb) to be installed.
+
+```
+python setup.py --command-packages=stdeb.command bdist_deb
+sudo dpkg -i deb_dist/python3-patacrep_4.0.0-1_all.deb
```

Comment:
Est-ce pertinent d'ajouter un fichier ``.deb`` à [la version en cours](/patacrep/patacrep/releases/tag/v4.0.0), ou est-ce que, vu que c'est du *quick and dirty*, il vaut mieux laisser les gens le construire eux-même ? Petite préférence pour la deuxième solution.